tests/fixtures/issues/issue-563.php

```diff
@@ -2691,12 +2691,12 @@
             'isModified' => false,
             'CssVersion' => '3',
             'AolVersion' => '0',
-            'Device_Name' => 'Marathon M3',
-            'Device_Maker' => 'Gionee',
+            'Device_Name' => 'M3',
```

Comment:
The device id of "M3" is pretty ambiguous, and different parsers disagree on what device it is.

In this case, I don't think it's the Gionee Marathon M3, as it seems that would be seen as "Gionee M3" and not just "M3". https://www.handsetdetection.com/properties/devices/Gionee/M3  Plus, as far as I can tell, the Gionee M3 is an Android 4.4+ device, while the HTM M3 appears to be Android 4.1/4.2: http://dpnhtech.com/htm-m3-5-0inch-mtk6572-dual-core-1-3ghz-smartphone-512m-ram-4gb-rom-5-0mp/

There's also the Tecno M3, but I believe that identifies itself as "Tecno M3" in useragents (I added this device and a test for it).